src/services/browserDataStorage/localStorage.ts

```diff
@@ -0,0 +1,11 @@
+export const setLocalItem = (key: string, value: any): void => {
+  localStorage.setItem(key, JSON.stringify(value));
+};
+
+export const getLocalItem = (key: string): any => {
+  return JSON.parse(localStorage.getItem(key)!);
```

Comment:
The "!" post-fix expression operator means the "localStorage.getItem(key)" value is not null/undefined, but as I see it may be null in case there is no such key in localstorage. Are you sure the "!" operator in necessary here?

Reply:
As the TypeScript error says, localStorage.getItem() can return either a string or null. 
JSON.parse() requires a string, so we should test the result of localStorage.getItem() before we try to use it.

I had 3 options:
1. JSON.parse(localStorage.getItem(key)!);
2. JSON.parse(localStorage.getItem(key) || '{}');
3. JSON.parse(localStorage.getItem(key) as string) or JSON.parse(localStorage.getItem(key) as any).

I chose the shortest one.

What do you say?)

Reply:
since it is common util and can be used in different places, and in future it may work not only with objects - I would recommend to do smth like:
```
export const getLocalItem = (key: string): any => {
  const result = localStorage.getItem(key):
  try {
    return JSON.parse(result);
  } catch {
    return result;
  }
  return JSON.parse(localStorage.getItem(key)!);
```

Reply:
Done!